client/src/components/FlexGrowContainer/FlexGrowContainer.styled.js

```diff
@@ -0,0 +1,10 @@
+import { styled } from "styled-components";
+
+const Div = styled.div`
+width: 100%;
+display: flex;
+justify-content: center;
+align-items: center;
+position: relative;
```

Comment:
what is the point of setting it to relative?

Reply:
I couldn't define the size of the artist image (the big one in the middle of the page), because I only used relative unites. So instead of using non-relative unites for the image, or the container of the image, I positioned the image with position absolute in its container (with position relative) - which solved the issue.